web/src/components/Navigation/Navigation.tsx

```diff
@@ -94,71 +88,96 @@ const Navigation = () => {
+                AlertStatusesEnum.Triaged
+              }`}
+              label="Alerts"
+            />
+          </Box>
+
+          <Box as="li" mb={2}>
+            <NavGroup
               active={isLogAnalysisNavigationActive}
               icon="log-analysis"
-              tooltipLabel="Log Analysis"
-              onClick={() =>
+              label="Log Analysis"
+              onSelect={() =>
                 setSecondaryNav(isLogAnalysisNavigationActive ? null : LOG_ANALYSIS_NAV_KEY)
               }
-            />
+            >
+              <LogAnalysisNavigation />
+            </NavGroup>
           </Box>
-          <Box as="li">
-            <NavIconButton
+          <Box as="li" mb={2}>
+            <NavGroup
               active={isComplianceNavigationActive}
               icon="cloud-security"
-              tooltipLabel="Cloud Security"
-              onClick={() =>
+              label="Cloud Security"
+              onSelect={() =>
                 setSecondaryNav(isComplianceNavigationActive ? null : COMPLIANCE_NAV_KEY)
               }
-            />
+            >
+              <ComplianceNavigation />
+            </NavGroup>
           </Box>
-          <Box as="li" mb="auto">
-            <NavIconButton
+          <Box as="li" mb={2}>
+            <NavGroup
               active={isSettingsNavigationActive}
-              icon="settings"
-              tooltipLabel="Settings"
-              onClick={() => setSecondaryNav(isSettingsNavigationActive ? null : SETTINGS_NAV_KEY)}
-            />
+              icon="settings-alt"
+              label="Settings"
+              onSelect={() => setSecondaryNav(isSettingsNavigationActive ? null : SETTINGS_NAV_KEY)}
+            >
+              <SettingsNavigation />
+            </NavGroup>
           </Box>
+
           <Box as="li" mt="auto">
-            <Link external href={PANTHER_DOCS_LINK} tabIndex={-1}>
-              <NavIconButton active={false} icon="docs" tooltipLabel="Documentation" />
-            </Link>
+            <Box
+              as={Link}
+              external
+              href={PANTHER_DOCS_LINK}
+              fontWeight="normal"
+              borderRadius="small"
+              tabIndex={-1}
+              px={4}
+              py={3}
+              fontSize="medium"
+              display="flex"
+              color="gray-50"
+              alignItems="center"
+              _hover={{
+                color: 'gray-50',
+                backgroundColor: 'navyblue-500',
+              }}
+            >
+              <Icon type="docs" size="medium" mr={3} />
+              <Box>Documentation</Box>
+            </Box>
           </Box>
+
           <Box as="li">
-            <Link as={RRLink} to={urls.account.support()} tabIndex={-1}>
-              <NavIconButton active={false} icon="help" tooltipLabel="Support" />
-            </Link>
+            <NavLink icon="help" label="Support" to={urls.account.support()} tabIndex={-1} />
```

Comment:
Adding a `tabIndex={-1}` here never makes it tab-able. I don't think this is what we want 

```suggestion
            <NavLink icon="help" label="Support" to={urls.account.support()}  />
```